cheatsheets/Session_Management_Cheat_Sheet.md

```diff
@@ -163,6 +163,49 @@ Typically, session management capabilities to track users after authentication m
 - Ensure entire cookie should be encrypted if sensitive data is persisted in the cookie
 - Define all cookies being used by the application, their name and why they are needed
 
+# HTML5 Web Storage API
+
+The Web Hypertext Application Technology Working Group (WHATWG) describes the HTML5 Web Storage APIs, localStorage and sessionStorage, as mechanisms for storing name-value pairs client-side.
+Unlike HTTP cookies, the contents of localStorage and sessionStorage are not automatically shared within requests or responses by the browser and are used for storing data client-side.
+
+## The localStorage API
+
+### Scope
+
+Data stored using the localStorage API is accessible by pages which are loaded from the same protocol, port and host.
+This provides similar access to this data as would be achieved by using the "secure" flag on a cookie, meaning that data stored from https could not be retrieved via http.
+
+### Duration
+
+Data stored using the localStorage API is persisted across browsing sessions, meaning that this data is susceptible to access by other system users. 
+
+### Use Case
+
+WHATWG suggests the use of localStorage for data that needs to be accessed across windows or tabs, across multiple sessions, and where large (multi-megabyte) volumes of data may need to be stored for performance reasons.
```

Comment:
It can be interesting to point to the warning about the concurrent RW access mentioned [here](https://html.spec.whatwg.org/multipage/webstorage.html#the-localstorage-attribute) at the bottom of the section:

 > The localStorage attribute provides access to shared state. This specification does not define the interaction with other browsing contexts in a multiprocess user agent, and authors are encouraged to assume that there is no locking mechanism. A site could, for instance, try to read the value of a key, increment its value, then write it back out, using the new value as a unique identifier for the session; if the site does this twice in two different browser windows at the same time, it might end up using the same "unique" identifier for both sessions, with potentially disastrous effects.

Reply:
For sure, i was thinking to only add a link but a summary, as you proposed, is good too 😃

Reply:
Added to Scope section